syntax = "proto3";

message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}

// The greeting service definition.
service HelloWorldService {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}